src/main/java/org/apache/commons/collections4/IterableUtils.java

```diff
@@ -804,6 +804,9 @@ public static boolean isEmpty(final Iterable<?> iterable) {
      * @return the number of elements contained in the iterable
      */
     public static int size(final Iterable<?> iterable) {
+        if (iterable == null) {
+            return 0;
+        }
```

Comment:
The return statement in this method `return IteratorUtils.size(emptyIteratorIfNull(iterable));` will return `0` anyway. But I think it's harmless to fail quicker, and avoids going through unnecessary steps. :+1: